### PR TITLE
[Changed] Update testing-library version

### PR DESCRIPTION
**Pull request by maurogestoso**
Updated the version of `testing-library` and added `user-event` in order to have a setup more similar to what we use.

Since the project doesn't have a `.prettierrc`, my VS Code formatted everything to the default rules. I thought that was OK for consistency.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,7 @@
-import React from 'react';
```
Comment (maurogestoso) [inline src/App.js:1]: Prettier

File: src/App.test.js [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
+import { render } from "@testing-library/react";
 
-import App from './App'
+import App from "./App";
 
 it('should show "Hi"', async () => {
-  const { getByText } = render(<App />)
+  const screen = render(<App />);
 
-  const text = getByText('Hi')
```
Comment (maurogestoso) [inline src/App.test.js:10]: Everything else is prettier changes

File: src/App.test.js [hunk 3]
```diff
@@ -1,13 +1,12 @@
-import React from 'react'
-import { render } from '@testing-library/react'
-import { shallow, mount } from 'enzyme'
```
Comment (maurogestoso) [inline src/App.test.js:3]: Do we want this here?

File: src/index.js [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+import * as serviceWorker from "./serviceWorker";
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(<App />, document.getElementById("root"));
 
 // If you want your app to work offline and load faster, you can change
 // unregister() to register() below. Note this comes with some pitfalls.
```
Comment (maurogestoso) [inline src/index.js:10]: Prettier

**Review by SalvaCarsi — APPROVED** [review]

File: src/App.test.js (hunk 3 above)
Comment (SalvaCarsi) [reply src/App.test.js:3]: The only reason I can think to keep it, is to have is as a placeholder for an applicant that do its test with enzyme.

**Pull request merged**